README.md

```diff
@@ -269,98 +401,90 @@ This is a "meta" gem, that simply depends on all the others. If you want to
     Package Registries->>Credentials Proxy: Creds are stripped by the Proxy
-    Credentials Proxy->>Dependabot Core: Dependabot Core never sees private registry credentials
+    Credentials Proxy->>Dependabot-Core: Dependabot-Core never sees private registry credentials
 ```
 
-This also means if Dependabot Core ever has a security vulnerability, those creds are still not at risk of being exposed.
-
-## Profiling
+This also means if Dependabot-Core ever has a security vulnerability, those creds are still not at risk of being exposed.
 
-You can profile a dry-run by passing the `--profile` flag when running it, or
-tag an rspec test with `:profile`. This will generate a
-`stackprof-<datetime>.dump` file in the `tmp/` folder, and you can generate a
-flamegraph from this by running:
-`stackprof --d3-flamegraph tmp/stackprof-<data or spec name>.dump > tmp/flamegraph.html`.
+# License and Project History
 
 ## Why is this public?
 
-As the name suggests, Dependabot Core is the core of Dependabot (the rest of the
+As the name suggests, Dependabot-Core is the core of Dependabot (the rest of the
 app is pretty much just a UI and database). If we were paranoid about someone
 stealing our business then we'd be keeping it under lock and key.
 
-Dependabot Core is public because we're more interested in it having an
+Dependabot-Core is public because we're more interested in it having an
 impact than we are in making a buck from it. We'd love you to use
 [Dependabot][dependabot] so that we can continue to develop it, but if you want
 to build and host your own version then this library should make doing so a
 *lot* easier.
 
-If you use Dependabot Core then we'd love to hear what you build!
+If you use Dependabot-Core then we'd love to hear what you build!
 
 ## License
 
 We use the License Zero Prosperity Public License, which essentially enshrines
 the following:
 
-- If you would like to use Dependabot Core in a non-commercial capacity, such as
+- If you would like to use Dependabot-Core in a non-commercial capacity, such as
   to host a bot at your workplace, then we give you full permission to do so. In
   fact, we'd love you to and will help and support you however we can.
 - If you would like to add Dependabot's functionality to your for-profit
-  company's offering then we DO NOT give you permission to use Dependabot Core
+  company's offering then we DO NOT give you permission to use Dependabot-Core
   to do so.
 
 All contributions to Dependabot Core implicitly transfer the IP of that contribution to
 GitHub, Inc. where it will be licensed the same way as above.
 
 ## History
 
-Dependabot and Dependabot Core started life as [Bump][bump] and
-[Bump Core][bump-core], back when Harry and Grey were working at
-[GoCardless][gocardless]. We remain grateful for the help and support of
+Dependabot and Dependabot-Core started life as [Bump][https://github.com/gocardless/bump] and
```

Comment:
Using `"Dependabot Core"` seemed ambiguous to me... we always refer to it as a single compound word, so I renamed it to `"Dependabot-Core"` everywhere...